network/node/accessor.go

```diff
@@ -70,7 +70,9 @@ func (a *Accessor) GetActiveNodeByShortID(shortID insolar.ShortNodeID) insolar.N
 }
 
 func (a *Accessor) GetActiveNodes() []insolar.NetworkNode {
-	return a.active
+	result := make([]insolar.NetworkNode, len(a.active))
```

Comment:
We should test copy behaviour.

Reply:
Mutator and Accessor tests will be done in a separate task